index.html

```diff
@@ -48,198 +65,251 @@ <h2 class="describe">I am a Software Developer</h2>
+          1500s, when an unknown printer took a standard dummy text.
+        </p>
 
-    <li>
-      <a class="button" href="#">See Project</a>
-    </li>
+        <ul class="languages">
+          <li class="language-icons">
+            <a href="#"><img src="images/css.png" alt="" /></a>
+          </li>
+          <li class="language-icons">
+            <a href="#"><img src="images/html.png" alt="" /></a>
+          </li>
+          <li class="language-icons">
+            <a href="#"><img src="images/bootstrap.png" alt="" /></a>
+          </li>
+          <li class="language-icons">
+            <a href="#"><img src="images/ruby.png" alt="" /></a>
+          </li>
+          <br />
+        </ul>
+
+        <li>
+          <a class="button" href="#">See Project</a>
+        </li>
+      </article>
+    </div>
   </section>
 
-  <section class="Professional">
-    <h5 class="heading5">Profesional Art Printing Data</h5>
-    <p class="art-words">
-      A daily selection of privately personalized reads; no accounts or sign-ups
-      required. has been the industry's standard
-    </p>
-    <a class="bgimage2"></a>
-    <ul class="f-item">
-      <li>
-        <a class="icon2" href="#"><img src="images/html1.png" alt="" /></a>
-      </li>
-      <li>
-        <a class="icon2" href="#"><img src="images/bootstrap1.png" alt="" /></a>
-      </li>
-      <li>
-        <a class="icon2" href="#"><img src="images/ruby1.png" alt="" /></a>
-      </li>
-    </ul>
 
-    <li>
-      <a class="button2" href="#">See Project</a>
-    </li>
-  </section>
+  <div class="Professional-cards">
+    <section class="Professional">
+      <h5 class="heading5">Profesional Art Printing Data</h5>
+      <p class="art-words">
+        A daily selection of privately personalized reads; no accounts or sign-ups
+        required. has been the industry's standard
+      </p>
+      <a class="bgimage2"></a>
+      <ul class="f-item">
+        <li>
+          <a class="icon2" href="#"><img src="images/html1.png" alt="" /></a>
+        </li>
+        <li>
+          <a class="icon2" href="#"><img src="images/bootstrap1.png" alt="" /></a>
+        </li>
+        <li>
+          <a class="icon2" href="#"><img src="images/ruby1.png" alt="" /></a>
+        </li>
+      </ul>
 
-  <section class="Professional">
-    <h5 class="heading5">Profesional Art Printing Data</h5>
-    <p class="art-words">
-      A daily selection of privately personalized reads; no accounts or sign-ups
-      required. has been the industry's standard
-    </p>
-    <a class="bgimage2"></a>
-    <ul class="f-item">
-      <li>
-        <a class="icon2" href="#"><img src="images/html1.png" alt="" /></a>
-      </li>
-      <li>
-        <a class="icon2" href="#"><img src="images/bootstrap1.png" alt="" /></a>
-      </li>
-      <li>
-        <a class="icon2" href="#"><img src="images/ruby1.png" alt="" /></a>
-      </li>
-    </ul>
+      <button class="button2">See Project</button>
```

Comment:
[OPTIONAL] 👍🏽 

- I'd like to kindly suggest that it will be great if you added hover effects to the following buttons in your portfolio 👇🏽 

| Your See Project button  |
| :---:   | 
|![seep](https://user-images.githubusercontent.com/99246359/208076857-abe909aa-246e-4ace-acb3-d2cf59f2dffb.PNG)  |

|Get my Resume button |
| :---:   | 
|![gmr](https://user-images.githubusercontent.com/99246359/208077020-8462793d-7a25-4ee5-b57d-32ef087eee74.PNG)  |

This is just for user experience and to make your website look prettier. 🌸